addons/mail/models/mail_message.py

```diff
@@ -761,8 +761,9 @@ def unstar_all(self):
 
         ids = [m.id for m in starred_messages]
         self.env['bus.bus']._sendone(self.env.user.partner_id, 'mail.message/toggle_star', {
-            'message_ids': ids,
             'starred': False,
+            'starred_counter': 0,
+            'data': [m.message_format()[0] for m in starred_messages],
```

Comment:
I think you can do something like that:
```suggestion
            'data': starred_messages.message_format(),
```